schema_editor/test/mock/resources/resources-mocks.js

```diff
@@ -130,11 +130,24 @@
                     'plural_label':'Bird Sightings',
                     'description':'Birds and their environments.',
                     'active':true
+                },
+                {
+                    'uuid': '99eeb841-31cf-4059-a408-070a0853c875',
+                    'current_schema': 'd71e6475-e328-41d3-8e1f-556ab4145129',
+                    'created': '2016-02-02T19:10:58.039863Z',
+                    'modified': '2016-02-02T19:10:58.039897Z',
+                    'label': 'Intervention',
+                    'plural_label': 'Interventions',
+                    'description': 'Actions to improve traffic safety',
+                    'active': true
                 }
             ]
         };
 
         var RecordType = RecordTypeResponse.results[0];
+
+        var SecondaryRecordType = RecordTypeResponse.results[2];
```

Comment:
Might make things a little more readable to move the result above bird sightings, so it will be the second thing.